packaging/flatpak/generate-sources.sh

```
@@ -0,0 +1,89 @@
+}
+
+make_asset_source \
+"https://api.github.com/repos/denoland/rusty_v8/releases/tags/v$(grep -oPm1 '"dest": "cargo/vendor/v8-\K[0-9]+([\.\d]+)' cargo-sources.json)" \
+"librusty_v8_release_{@}-unknown-linux-gnu.a.gz" \
+"file" \
+"dest-filename" \
+"librusty.a.gz" \
+"librusty.json"
+
+make_asset_source \
+"https://api.github.com/repos/DioxusLabs/dioxus/releases/tags/v$(grep -oPm1 'cargo install dioxus-cli@\K[0-9]+([\.\d]+)' $CUR_DIR/.github/workflows/release.yml)" \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)